internal/testplanet/planet.go

```diff
@@ -403,6 +403,7 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 					Email:  prefix + "@example.com",
 					Wallet: "0x" + strings.Repeat("00", 20),
 				},
+				QueryTimeout: 60 * time.Second,
```

Comment:
I think this should be `pingbackTimeout`

Reply:
how much smaller? also, do you know why the overlay tests are failing now? it seems unrelated.

Reply:
not sure how much smaller... I guess we can leave it at 60s for now until we figure out a better number. Not sure about the tests.

Reply:
last time I checked (prior to hammer merge) it could take up to 2 minutes to receive a node-info response back. so there will be node-info potentially missed if we make this smaller.